unit-tests/src/test/java/org/eclipse/collections/impl/collector/SerializableLongSummaryStatisticsTest.java

```diff
@@ -29,6 +30,7 @@ public void valuesEqual()
         Assert.assertTrue(with.valuesEqual(without));
     }
 
+    @Ignore
```

Comment:
Instead of ignoring or deleting, we could add an Assume statement using the system property for the version of Java.